proto/prysm/v1alpha1/BUILD.bazel

```diff
@@ -45,111 +45,314 @@ proto_library(
-        "SignedBeaconBlockCapella",
-        "BeaconBlockDeneb",
-        "BeaconBlockBodyDeneb",
-        "SignedBeaconBlockDeneb",
-        "BeaconBlockElectra",
-        "BeaconBlockElectra",
-        "SignedBeaconBlockElectra",
-        "SignedBlindedBeaconBlockCapella",
-        "BlindedBeaconBlockCapella",
-        "BlindedBeaconBlockBodyCapella",
-        "SignedBlindedBeaconBlockDeneb",
-        "BeaconBlockContentsDeneb",
-        "SignedBeaconBlockContentsDeneb",
-        "BlindedBeaconBlockDeneb",
-        "BlindedBeaconBlockBodyDeneb",
-        "SignedBlindedBeaconBlockElectra",
-        "BlindedBeaconBlockElectra",
-        "BlindedBeaconBlockBodyElectra",
-        "SyncAggregate",
-        "SyncCommitteeMessage",
-        "SyncCommitteeContribution",
-        "ContributionAndProof",
-        "SignedContributionAndProof",
-        "BeaconBlocksByRangeRequest",
-        "BlobSidecarsByRangeRequest",
         "ENRForkID",
-        "MetaDataV0",
-        "MetaDataV1",
         "Status",
         "AggregateAttestationAndProof",
-        "AggregateAttestationAndProofElectra",
         "Attestation",
-        "AttestationElectra",
         "AttestationData",
         "AttesterSlashing",
-        "AttesterSlashingElectra",
         "BeaconBlock",
         "BeaconBlockHeader",
         "Checkpoint",
         "Deposit",
         "Eth1Data",
         "IndexedAttestation",
-        "IndexedAttestationElectra",
         "ProposerSlashing",
         "SignedAggregateAttestationAndProof",
-        "SignedAggregateAttestationAndProofElectra",
         "SignedBeaconBlock",
         "SignedBeaconBlockHeader",
         "SignedVoluntaryExit",
         "Validator",
         "VoluntaryExit",
-        "ContributionAndProof",
-        "SignedContributionAndProof",
         "DepositMessage",
         "Fork",
         "ForkData",
         "HistoricalBatch",
         "Status",
         "BeaconState",
-        "BeaconStateAltair",
-        "BeaconStateBellatrix",
-        "BeaconStateCapella",
-        "BeaconStateDeneb",
-        "BeaconStateElectra",
         "SigningData",
+        "PowBlock",
+    ],
+)
+
+ssz_gen_marshal(
+    name = "ssz_generated_altair",
+    go_proto = ":go_proto",
+    out = "altair.ssz.go",
+    includes = [
+        "//consensus-types/primitives:go_default_library",
+        "//proto/engine/v1:go_default_library",
+        "//math:go_default_library",
+    ],
+    objs = [
+        "SignedBeaconBlockAltair",
+        "BeaconBlockAltair",
+        "BeaconBlockBodyAltair",
+        "BeaconStateAltair",
+        "SyncAggregate",
+        "SyncCommitteeMessage",
+        "SyncCommitteeContribution",
+        "ContributionAndProof",
+        "SignedContributionAndProof",
+        "SyncAggregate",
         "SyncCommittee",
         "SyncAggregatorSelectionData",
-        "PowBlock",
-        "SignedValidatorRegistrationV1",
-        "ValidatorRegistrationV1",
-        "Withdrawal",
+    ],
+    exclude_objs = [
+        "Attestation",
```

Comment:
These excludes look like prior fork objs. Perhaps these can be better arranged like 
```
ssz_gen_marshal(
    name = "ssz_generated_altair",
    ...
    objs = ssz_altair_objs,
    excludes = ssz_phase0_objs,
)
 ```